README.md

```diff
@@ -1389,12 +1392,58 @@ Like API Gateway, Zappa can automatically provision ALB resources for you.  You'
         // And here, a list of security group IDs, eg. 'sg-fbacb791'
     ]
 }
-```
```

Comment:
The deletion of this line broke the next session of the readme. The sessions _Endpoint Configuration_ and _Example Private API Gateway configuration_ are being displayed as part of this block of code.

Since this was already merged, I'll open a new issue & PR fixing it